orb_models/finetune_utilities/steps.py

```diff
@@ -0,0 +1,168 @@
+from typing import List, Optional, Union, cast
+
+import torch
+import tqdm
```

Comment:
Let's remove tqdm and just print a step update

Reply:
Would there be any issue with `tqdm`? Currently it is showing the progress nicely when launching the run with command line. Printing steps would end up with too may terminal output?